operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1236,35 +1245,49 @@ spec:
                     - name: STRIMZI_OPERATION_TIMEOUT_MS
                       value: "300000"
                     - name: STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE
-                      value: registry.redhat.io/amq-streams/kafka-37-rhel9@sha256:3e41f47d21c030b4f0f29b96a71e9952166c5abbf0539e9ae0247f28b56bb735
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.annotations['kafka-current-image']
```

Comment:
This makes me happy 😁 It clearly tells me why we have a particular kafka version here and means we only need to update one reference to get everything updated.